src/exporto0o.ts

```diff
@@ -163,9 +163,11 @@ export async function exportToOo(
   }
 
   try {
-    await exec(cmd, { cwd: variables.currentDir, env: { TEXINPUTS: joinEnvPath(`${textemplateDir}`, './') } }); 
+    const { exec } = require('child_process');
+    const path = require('path');
     console.log(`[${plugin.manifest.name}]: export command: ${cmd}`);
-    await exec(cmd, { cwd: variables.currentDir });
+    // orginal `$TEXINPUTS` should be empty, so ignore it.
+    await exec(cmd, { cwd: variables.currentDir, env: { TEXINPUTS: joinEnvPath(`${textemplateDir}`, `${process.env.TEXINPUTS}`) } }); 
```

Comment:
The
```js
`${textemplateDir}`
``` 
is Javascript [string template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), when only one variable, you can use `textemplateDir` directly.

Reply:
Thanks - should be fixed now!